### DISCUSSION TITLE
Add b/kb/mb/gb/tb/pb/eb option to file size interface

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4945] [Feature Requests]
![image](https://user-images.githubusercontent.com/8030134/114222959-d0fda400-99a1-11eb-9c8c-c2778d6e9031.png)

**Comment by cdwmhcc** [comment]
@rijkvanzanten @benhaynes

**Reply by azrikahar** [reply]
Hi, just to clarify, is the current issue with the filesize display not showing simplified value (kb/mb etc)? such as this:

![chrome_SpAXKXLvhV](https://user-images.githubusercontent.com/42867097/137274017-8293bbc9-298e-4801-b539-eb914e144a31.png)

Or do you mean somewhere else? Though to be fair, these should probably be formatted regardless, so will make a quick PR for it.

**Reply by azrikahar** [reply]
Seems like this was previously clarified back in https://github.com/directus/directus/discussions/4690#discussioncomment-534444, and it is correct in that we only have display at the moment. 

With that said, added a PR to improve the display: #8786. Hopefully this removes the need of having an interface to enforce a particular conversion, unless there's a use case for it 🤔

**Reply by benhaynes** [reply]
Thanks, @azrikahar!! Awesome improvements to the Filesize Display!

Not sure if we have a Filesize Interface, but it would be a nice one to have at some point. I assume it will mostly be disabled (entered on upload/create), but I like the general design posted above.

**Reply by cdwmhcc** [reply]
> Seems like this was previously clarified back in [#4690 (reply in thread)](https://github.com/directus/directus/discussions/4690#discussioncomment-534444), and it is correct in that we only have display at the moment.
> 
> With that said, added a PR to improve the display: #8786. Hopefully this removes the need of having an interface to enforce a particular conversion, unless there's a use case for it 🤔

It is the default interface in v8. It disappeared in v9.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
